Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/caffe_impl.cpp

```diff
@@ -11,8 +11,9 @@
 #include "tbb/parallel_for_each.h"
 #include "../classification_result.hpp"
 
-// Based on the rs-dnn example
-
+/// <summary>
+/// This struct creates a caffe-based MobileNet SSD object recognition network
```

Comment:
MobileNet SSD? Is that correct? Should not just be a caffe-based object ....
Doesn't it depend on the model files used in the Constructor?